content/open-source/concepts/bootstrap/bootstrap.md

```diff
@@ -23,7 +23,7 @@ CatalogContent:
   - 'paths/computer-science'
 ---
 
-[**Bootstrap**](https://getbootstrap.com/) is an open-source [CSS](https://www.codecademy.com/resources/docs/css) [framework](https://www.codecademy.com/resources/docs/general/framework) created to help style web pages with mobile-first considerations in mind. It was created internally at Twitter in 2011 as a way to solve issues with design consistency across their engineering team. Since its creation, Bootstrap has become one of the most commonly used tools on web pages today.
+[**Bootstrap**](https://getbootstrap.com/) is an open-source [CSS](https://www.codecademy.com/resources/docs/css) [framework](https://www.codecademy.com/resources/docs/general/framework) created to help style web pages with mobile-first considerations in mind. It was created internally at X in 2011 as a way to solve issues with design consistency across their engineering team. Since its creation, Bootstrap has become one of the most commonly used tools on web pages today.
```

Comment:
```suggestion
[**Bootstrap**](https://getbootstrap.com/) is an open-source [CSS](https://www.codecademy.com/resources/docs/css) [framework](https://www.codecademy.com/resources/docs/general/framework) created to help style web pages with mobile-first considerations in mind. It was created internally at Twitter (now X) in 2011 as a way to solve issues with design consistency across their engineering team. Since its creation, Bootstrap has become one of the most commonly used tools on web pages today.
```